data/json/items/ammo/300BLK.json

```diff
@@ -1,45 +0,0 @@
-[
```

Comment:
Was this file deleted accidentally?

Reply:
I spoke to people on the discord about the problem that someone else had modified 300BLK extensively right around same time as I was adding apoc_price to all the ammo jsons.  They suggested that since it was my first time contributing that I delete 300BLK from my PR request so my changes don't go conflict with the already mainlined changes and then I can come back and add apoc price specifically to that one file.  I'm worried that I made my PR say it should delete 300BLK file entirely from mainstream.  Please let me know which one I've done and if I can fix it?

Reply:
I see. Yes, I guess there was some kind of miscommunication and by "deleting file X from pr" these people suggested "not doing changes in file X". I will restore the file.

Reply:
Just check whether prices are okay in this file now and we are good to go.

Reply:
Ok I resolved the multiple price issue.  Thank you!